src/ucs/configure.m4

```diff
@@ -218,10 +218,12 @@ case ${host} in
                   [[#include <stdint.h>]],
                   [[uint64_t tmp; asm volatile("mrs %0, cntvct_el0" : "=r" (tmp));]])],
                   [AC_MSG_RESULT([yes])]
-		  [AC_DEFINE([HAVE_HW_TIMER], [1], [high-resolution hardware timer enabled])],
-		  [AC_MSG_RESULT([no])]
-		  [AC_DEFINE([HAVE_HW_TIMER], [0], [high-resolution hardware timer disabled])]
-                 );;
+                  [AC_DEFINE([HAVE_HW_TIMER], [1], [high-resolution hardware timer enabled])],
+                  [AC_MSG_RESULT([no])]
+                  [AC_DEFINE([HAVE_HW_TIMER], [0], [high-resolution hardware timer disabled])],
+                  [AC_MSG_RESULT([no - cross-compiling detected])]
+                  [AC_DEFINE([HAVE_HW_TIMER], [0], [high-resolution hardware timer disabled])]
+                  );;
```

Comment:
indentation/scope of [ ] seems weird
what are the then/else parts here?

Reply:
@yosefe looks fine to me. Yes/No/Cross-compile. There is no need for additional indentation.

Reply:
let's add a [ ] scope for each such section, as in other places

Reply:
Extra [] causes error with autoconf 2.69. It also not needed.

Reply:
i mean something like this: https://github.com/openucx/ucx/blob/master/config/m4/compiler.m4#L425